liquibase/liquibase.github.com
# New change layout as in proposal: https://forum.liquibase.org/#Topic/49382000002120009

## What I Did
Updated:
- changelog_parameters.md:
  - Fix property name, value definition, clarify
  - Added yaml sample
  - Fixed parameter reading order
- Fix include.relativeToChangelogFile

Added:
- changeSet.logicalFilePath, runOrder, created, ignore + Note on file path
- Databasechangelog.objectQuotingStrategy
- parameter definition in defaults properties file
- Overview table in /documentation/changes/index.md with the available changes
- command line commands: calculateCheckSum, executeSql

Generator changes:
- See: https://forum.liquibase.org/#Topic/49382000002120009
- Added generic description for some properties (encoding/dbms/relativeToChangelogFile) if not set in the DatabaseChangeProperty (A more generic solution might to use separate interfaces for the changes in the core. Or use a simple properties file for the generator)
- More detailed samples: Some comes from the core + some fixed in the generator

## How To Test It
See [generated sample](https://b-gyula.github.io/liquibase-doc/documentation/changes/) 

(Some property description will be available, after [PR 1002](https://github.com/liquibase/liquibase/pull/1002), but there is no binary dependency)
